### DISCUSSION TITLE
Add loading spinner when working on tables.

### DISCUSSION THREAD

**Discussion by j-vitali** [filamentphp/filament#13027] [Ideas]
Hi guys, will be nice if there's a `loading` spinner, or maybe just a span "_loading..._" so user can see that changes were applied.

In this scenario, i wanted to see all records, and it's not so intuitive without the loading..

<img width="1437" alt="Screenshot 2024-05-31 alle 10 54 16" src="https://github.com/filamentphp/filament/assets/161858414/493ae830-0d5a-4843-95ce-8fd5eeb13ea9">

**Comment by dissto** [comment]
Probably not what you had in mind but take a look at [spa mode](https://filamentphp.com/docs/3.x/panels/configuration#spa-mode)

> ... and a loading bar for longer requests

**Reply by j-vitali** [reply]
basically yes and not... i mean, if they did it for spa, can do also for normal pages....
I mean only a loading on table... but of course a laoding in all separated components can be awesome
